src/components/Cell/Cell.js

```diff
@@ -0,0 +1,70 @@
+
+import React from "react";
+import flag from "/Users/itaimondshine/SampleJS/new-app/src/components/Cell/flag.png"
+
+
+let RenderCell = function(props) {
+  let cell =  function ()  {
```

Comment:
This function isn't needed. Try to combine your conditions in one `return` statement.